docs/jit.md

```diff
@@ -49,9 +49,9 @@ There are a number of ways to disable the JIT:
 
 - Specify `-Djava.compiler=NONE` on the command line.
 - Specify `-Xint` on the command line, which turns off the JIT and AOT compiler. To eliminate problems with one or the other you can turn these compilers off selectively with the `-Xnojit` and `-Xnoaot` options.
-- Call the `java.lang.Compiler` API programmatically.
+- Call the `java.lang.Compile` API programmatically.
```

Comment:
Compiler is correct.
http://w3.hursley.ibm.com/java/docs/java8/api/java/lang/Compiler.html

Reply:
Tsk ... I've made bad notes here. Cancelling this PR. Sorry!